.travis.yml

```diff
@@ -19,7 +19,7 @@ addons:
       - rsyslog
   mariadb: "10.0"
 
-sudo: false
+sudo: true
```

Comment:
This will cause us to be unable to use Travis' container based infrastructure which will considerably slow down tests, does Travis set permissions such that only the root/mysql user can read the error log?